sdk/resourcemanager/azure-resourcemanager-cdn/pom.xml

```diff
@@ -45,6 +45,7 @@
 
       --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```

Comment:
why?

Reply:
Some Java Beans related to `MetricsResponse` have been deleted or renamed. For details, please refer to the following URL：https://dev.azure.com/azure-sdk/public/_build/results?buildId=3162656&view=logs&jobId=b70e5e73-bbb6-5567-0939-8415943fadb9&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=ba3cf43a-b2bf-50f9-f95f-93049ca44ad1

Reply:
So we need to know why it been deleted. codegen? service? if service did they get approval?

Reply:
I compared the swagger description of `package-2023-05` and `package-2021-06`. After analyzing the comparison results, I thought that the service had changed.

Reply:
Could you find the place of the change? Similar to that in cosmos.

You can check history https://github.com/Azure/azure-rest-api-specs/commits/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01 and then check the breaking changes

Reply:
Important changes in version `package-2023-05`:
Added items
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L85
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L240
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L291
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L4208
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L4457
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L5029
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L5835
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L5857
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L6069
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L6086
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L6298

Remove services
"/subscriptions/{subscriptionId}/providers/Microsoft.Cdn/validateSecret"

Reply:
Which part of it relates to the breaks?

Reply:
The main impacts come from the following four parts:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L5836
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L5857
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L6069
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/afdx.json#L6086

Reply:
Great. Thanks.
So the reason is that service added name to a previous unnamed enum.

@XiaofeiCao @haolingdong-msft 
Since convenience code does not use these, maybe we just go skip?

Reply:
Need to keep `revapi.skip` ?

Reply:
Agree. If no convenience code use it, we can keep the break. 

@v-hongli1 Please add a Breaking Changes section in changelog.md mentioning this breaking change.